Adds read/write permission

Rafase282 opened the pull request:
Closes #13 

Implements permission mechanism using `.env` files.

## Read Permission

Used for Podio API calls to retrieve information only.

### Write Permission

Used for Podio API calls to update information.
*Example:* set command.

## Other Changes

* Removes orphan log from test.
* Disables complexity check for eslint.

Rafase282 committed:
Adds read/write permission

flyblackbox reviewed (approved):
Not sure why you deleted some of those lines but the read / write permissions config feature looks great!

Rafase282 commented:
@flyblackbox  I removed those lines because I'm planing on ditching the app.<service>.<function> but while I was working on that I broke something so I reverted. However, for the commands, it is not really needed so I decided to take it out right there.
